Menu/Menu.js

```diff
@@ -1,10 +1,18 @@
-
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
-}
+  // Toggle the "menu--open" class on your menu refence.
+  if(menu.classList.toggle("menu--open")){
+    TweenMax.fromTo(menu, 0.5, { left: -350 }, { left: 0});
+  } else {
+    TweenMax.fromTo(menu, 0.5, { left: 0 }, { left: -350});
+  }
+};
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector(".menu");
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector(".menu-button");
 // Using your menuButton reference, add a click handler that calls toggleMenu
+
+menuButton.addEventListener("click", () => {
```

Comment:
since this toggle menu function makes no use of the THIS keyword, you could put the binding name directly into this event listener >> 
```
menuButton.addEventListener("click", toggleMenu)
```